cityofaustin/vision-zero
# Create database table to store crash-component lookup table

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19454

## Testing

**URL to test:** 
Local

**Steps to test:**
Check out this branch, start your local db, run the migrations and apply metadata. Test running the down migrations too, inspect the table. Test the down migrations. 

---

#### Ship list
